src/rosconsole/impl/rosconsole_glog.cpp

```diff
@@ -97,6 +97,10 @@ void register_appender(LogAppender* appender)
   rosconsole_glog_appender = appender;
 }
 
+void deregister_appender(){
```

Comment:
Please follow the same style of the file: putting the `{` in a new line.

Same below.